packages/pouchdb-replication/src/index.js

```diff
@@ -4,6 +4,24 @@ import sync from './sync';
 function replication(PouchDB) {
   PouchDB.replicate = replicate;
   PouchDB.sync = sync;
+
+  Object.defineProperty(PouchDB.prototype, 'replicate', {
```

Comment:
ah yeh I forgot its tricky in that case, yeh no bother I cant think of anything cleaner off the top of my head and wasnt a major concern, mostly minor code style, cheers